src/client/assets/main.coffee

```diff
@@ -3,6 +3,7 @@
 # can break this up into self-initializing app-level asset packages like Force.
 #
 
+require('regenerator-runtime/runtime')
```

Comment:
I think regenerator is only needed here -- that or only in `src/index.js`. Can remove in other places.

Reply:
Did some experiments moving/removing, but we definitely need it here otherwise it fails to load the bundle and throws errors for missing regeneratorRuntime in the browser console.